### DISCUSSION TITLE
Add a direct link/method of navigation from one course to the next within a path.

### DISCUSSION THREAD

**Discussion by Advaitmenon1106** [TheOdinProject/theodinproject#4010] [Ideas]
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this issue follows the `brief description of request` format, e.g. `Add dark mode to website`

The following checkbox is OPTIONAL:

-   [x] I would like to be assigned this issue to work on it

<hr>

**1. Description of the Feature Request:**
 FullStack JavaScript (Resume button) --> JavaScript (Open Course button) {this is the convoluted way}

My suggestion: Add a button or a link to the next course from the previous course in either the last lesson of the previous course, or the table of contents of the previous course, in order to make things go a bit more smoothly or efficiently.

-->

**2. Acceptance Criteria:**

**3. Additional Information:**

Something like this...

![Screenshot from 2023-07-14 19-28-32](https://github.com/TheOdinProject/theodinproject/assets/94791117/d03645ea-ee3c-4bb4-94a8-d89d6a076bc6)

And/or this...

![Untitled](https://github.com/TheOdinProject/theodinproject/assets/94791117/a3bf917e-74c5-45cc-b542-de3f6051f59e)

**Comment by Asartea** [comment]
I think there is different merit to this: the current UX of clicking "complete lesson", and then needing to back out to the path level (which isn't necessarily easy either) and then selecting a new course is weird, and also conflicts with how we handle say choosing a new path after Foundations

**Comment by Advaitmenon1106** [comment]
> I think there is different merit to this: the current UX of clicking "complete lesson", and then needing to back out to the path level (which isn't necessarily easy either) and then selecting a new course is weird, and also conflicts with how we handle say choosing a new path after Foundations

Yes! That too

**Comment by CouchofTomato** [comment]
@KevinMulhern Any thoughts on this mate.
